nifi-nar-bundles/nifi-standard-services/pom.xml

```diff
@@ -15,6 +15,18 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <dependencies>
```

Comment:
This dependencies section shouldn't be needed here. This is the parent pom for all of the standard services.

Reply:
Done.